esg_node.py

```diff
@@ -385,25 +385,13 @@ def clear_tomcat_cache():
         logger.exception(error)
 
 def remove_unused_esgf_webapps():
-    '''Hard coded to remove node manager, desktop and dashboard'''
+    '''Hard coded to remove node manager'''
```

Comment:
I honestly can't remember why this function to remove the webapps was written.  It might be worth deleting this function altogether and leaving the node manager in place too.